wolfhsm/wh_server.h

```diff
@@ -2,18 +2,51 @@
 #define WOLFHSM_WH_SERVER_H_
 
 #include "wolfhsm/wh_comm.h"
+#include "wolfhsm/wh_common.h"
+#include "wolfhsm/common.h"
+
+#ifndef WOLFSSL_USER_SETTINGS
+    #include "wolfssl/options.h"
+#endif
+#include "wolfssl/wolfcrypt/settings.h"
+#include "wolfssl/wolfcrypt/random.h"
+#include "wolfssl/wolfcrypt/curve25519.h"
 
 #if 0
 #include "wolfhsm/nvm.h"
 #include "wolfhsm/nvm_remote.h"
 #endif
 
+typedef struct NvmMetaData {
+    uint16_t id;
+    uint32_t flags;
+    uint16_t len;
+    uint8_t label[WOLFHSM_NVM_LABEL_LEN];
+    uint8_t confDigest[WOLFHSM_DIGEST_STUB];
+#ifdef WOLFHSM_SHE_EXTENSION
+    uint32_t count;
+#endif
+} NvmMetaData;
```

Comment:
Why are we defining a server-specific meta data struct here? Isn't [whNvmMetaData ](https://github.com/wolfSSL/wolfHSM/blob/main/wolfhsm/wh_common.h#L76)already defined in wh_common.h? I don't think we want multiple clones of this floating around